src/main/java/com/spotifyapi/dto/TokensDTO.java

```diff
@@ -1,8 +1,6 @@
 package com.spotifyapi.dto;
 
-import lombok.AllArgsConstructor;
-import lombok.Getter;
-import lombok.Setter;
+import lombok.*;
```

Comment:
don't use * in import

Reply:
Changed on: 
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;